barretenberg/cpp/pil/vm2/context.pil

```diff
@@ -1,17 +1,79 @@
 // This is a virtual gadget, which is part of the execution trace.
 namespace execution;
 
+    // Useful to define some opcodes within this pil file
+    // Constrained to be boolean by execution instruction spec table
+    pol commit call_sel;
+    pol commit static_call_sel;
+    pol CALL = call_sel + static_call_sel; // Guaranteed to be mutually exclusive
+    // CALL & precomputed.first_row are NAND
+    CALL * precomputed.first_row = 0;
+
+    // Context columns
     pol commit context_id;
+    pol commit parent_id;
     pol commit pc;
+    pol commit next_pc;
     pol commit msg_sender;
     pol commit contract_address;
-    
-    pol commit is_static;
-    is_static * (1 - is_static) = 0;
+    pol commit is_static; // Constrained boolean by tx trace
```

Comment:
Isn't this only true for contexts from enqueued calls, but not for nested calls?

Reply:
ah yep - updated the comment to mention that nested calls are constrained by `#[NEXT_IS_STATIC]`